### ISSUE TITLE
update_packages throws error

### ISSUE THREAD

**Issue by jonathan-g** [r-lib/remotes#232]
Both with the CRAN version and with the github version, remotes::update_packages() throws an error every time I call it:
```
> update_packages()
Skipping 1 packages not available: translations
Error in FUN(X[[i]], ...) : is.remote(remote) is not TRUE
```
```
> sessionInfo()
R version 3.5.1 (2018-07-02)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 17134)

Matrix products: default

locale:
[1] LC_COLLATE=English_United States.1252  LC_CTYPE=English_United States.1252    LC_MONETARY=English_United States.1252 LC_NUMERIC=C                          
[5] LC_TIME=English_United States.1252    

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] remotes_2.0.2

loaded via a namespace (and not attached):
[1] compiler_3.5.1  tools_3.5.1     curl_3.2        packrat_0.4.9-3
```

translations is the translations package in the base R distribution (it's in C:\Program Files\R\R-3.5.1\library\translations). Googling tells me that things sometimes have a hard time dealing with translations because the package does not have a namespace.

Also, the DESCRIPTION file for translations does not include a source URL.
```
Package: translations
Version: 3.5.1
Title: The R Translations Package
Author: R Core Team and contributors worldwide
Maintainer: R Core Team <R-core@r-project.org>
Description: Compiled translations of messages.
License: Part of R 3.5.1
```

**Comment by jimhester** [comment]
I don't think the error is coming from translations but from a different package.

e.g. I think you will find `remotes::update_packages("translations")` works, (with a `Skipping 1 packages not available` message).

Try running the following and past the full output so we can narrow down what is happening.

```r
# enable more debugging information
trace(remotes:::is.remote, quote(str(x)))

# This will likely error
remotes::update_packages()

#After the error run traceback
traceback()
```

Also if you are able to determine the package that is causing the error the output of `packageDescription("failingpkg")` would be useful.

**Comment by jonathan-g** [comment]
I'm afraid that I can't figure out which package is causing the error.
```
> trace(remotes:::is.remote, quote(str(x))) 
Tracing function "is.remote" in package "remotes (not-exported)" 
[1] "is.remote" 
> remotes::update_packages() 
Skipping 1 packages not available: translations 
Tracing is.remote(remote) on entry   
NULL 
Error in FUN(X[[i]], ...) : is.remote(remote) is not TRUE 
> traceback() 
8: stop(simpleError(msg, call = sys.call(-1))) 
7: stopifnot(is.remote(remote)) 
6: FUN(X[[i]], ...) 
5: vapply(remotes, install_remote, ..., FUN.VALUE = character(1)) 
4: install_remotes(object$remote[!object$is_cran & behind], dependencies = dependencies,
         upgrade = upgrade, force = force, quiet = quiet, build = build,
         build_opts = build_opts, repos = repos, type = type, ...) 
3: update.package_deps(pkgs, dependencies = dependencies, upgrade = upgrade,
         force = force, quiet = quiet, build = build, build_opts = build_opts,
         repos = repos, type = type, ...) 
2: update(pkgs, dependencies = dependencies, upgrade = upgrade,
         force = force, quiet = quiet, build = build, build_opts = build_opts,
         repos = repos, type = type, ...) 
1: remotes::update_packages()
```
Here's what I tried to do to figure out what the package was that caused the fault:
```
packages <- utils::installed.packages()[, "Package"] 
> pkgs <- remotes::package_deps(packages) 
> ug <- remotes:::upgradable_packages(pkgs, upgrade = "ask") 
> ug 
Not on CRAN ----------------------------  package
      installed available is_cran remote  translations 3.5.1     NA        TRUE    CRAN
```
